### PR TITLE
Chapter 7: Sign up

### PR DESCRIPTION
**Pull request by rashidfirhat98**

### CODE REVIEW

**Comment by rashidfirhat98** [comment]
ready

**Comment by rashidfirhat98** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (saobv-2223) [inline app/assets/stylesheets/users.scss]: remove file that never used

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render :new
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb:17]: add flash for this case

Comment (rashidfirhat98) [reply app/controllers/users_controller.rb:17]: done

**Comment by rashidfirhat98** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -90,3 +90,89 @@ footer ul li {
...
+  margin-top: 15px;
+}
+
+/* forms */
+input, textarea, select, .uneditable-input {
+  border: 1px solid $mediumlightgrey;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
+  }
```
Comment (saobv-2223) [inline app/assets/stylesheets/custom.scss]:
```suggestion
}
```

**Review by saobv-2223 — COMMENTED** [review]
fix comment and merge

**Pull request merged**